onyo/lib/commands.py

```diff
@@ -319,15 +320,19 @@ def _edit_asset(inventory: Inventory,
         subprocess.run(f'{editor} {quote(str(tmp_path))}', check=True, shell=True)
         operations = None
         try:
-            asset = DotNotationWrapper(get_asset_content(tmp_path))
-            if 'is_asset_directory' in asset.keys():
+            tmp_asset = DotNotationWrapper(get_asset_content(tmp_path))
+            if 'onyo.is.directory' in tmp_asset.keys():
                 # special case
-                # 'is_asset_directory' currently is the only modifiable, reserved key.
+                # 'onyo.is.directory' currently is the only modifiable, reserved key.
```

Comment:
Semi-allowed at this point. I have neither fully implemented setting other pseudo-keys nor tested it. So, currently it would be settable via `--keys` but not from within the editor (like before). IOW: It's already stripped to not have it listed in the error message on setting disallowed keys, but if you set it in the edited YAML it would be ignored.

So, just a consequence of the intermediate state right now. Comment here is in-line with the code it's about. But that code will change.